### PR TITLE
feat: add publicPath options

### PR DESCRIPTION
**Pull request by QingWei-Li**
See https://github.com/webpack-contrib/worker-loader/#publicpath

### CODE REVIEW

File: src/index.js
Comment (developit) [inline src/index.js]:
```suggestion
				worker.url = `${JSON.stringify(options.publicPath + worker.file)}`;
```

**Comment by xzilja** [comment]
I was just about to submit similar PR with added "name" field (will submit this pr once this lands). @developit it looks like failing ci is not related to the change / rather some package. Do you have any idea on what is causing it, I'd be up to submit commit to fix this, as this PR enables workerize loader to be used in next.js https://github.com/developit/workerize-loader/issues/80

**Comment by Poky85** [comment]
Tested and confirm that `publicPath` option allows Next.js integration with following `next.config.js`:

```javascript
module.exports =  {
  webpack: (config, params) => {
    config.module.rules.push({
      test: /\.worker\.(js|ts)$/,
      loader: 'workerize-loader',
      options: {
        name: 'static/[hash]',
        publicPath: '/_next/'
      },
    });
    config.output.globalObject = 'self';

    return config
  }
};
```

**Comment by developit** [comment]
Apologies for the super slow merge here! I'm fixing the CI, you're right about it being totally unrelated.

**Review by developit — APPROVED** [review]

**Comment by xzilja** [comment]
@QingWei-Li any chance you could merge this please?

**Comment by QingWei-Li** [comment]
@IljaDaderko I'm not a maintainer. We need @developit to merge.

**Comment by mrdivyansh** [comment]
@developit any chance you could merge this please?

**Comment by mysticaltech** [comment]
Any updates on the merge, please folks, make this move, it's hard enough for newbies like me to get started with next js, I am moving an app away from CRA and am encountering a Worker undefined error. Merging this should hopefully fix it.

**Comment by developit** [comment]
Released in [1.3.0](https://github.com/developit/workerize-loader/releases/tag/1.3.0).

**Pull request merged**
